nginx.conf

```diff
@@ -1,11 +1,11 @@
 user  nginx;
-worker_processes  1;
+worker_processes  auto;
```

Comment:
Betyr dette at det blir en pr kjerne tilgjengelig?

Det kan kanskje være litt skummelt om den blir throttla før av kubernetes? Bør vi justere requests/limits i kubernetes og evt?

Reply:
Kan godt droppe dette. Så lenge vi kjører i k8s er 1 sikkert like bra.

Reply:
Vi kan sikkert sette den til et høyere tall og bumpe limiten i deploy da.
Vi kan prøve med 2 for å se?